src/config/build_config.py

```diff
@@ -1,3 +1,6 @@
+# Please only enable it if you are preparing an EIA related PR
+# Do remember to revert it back before merging any PR (including EIA dedicated PR)
+ENABLE_EI_MODE = False
```

Comment:
Should I call it "DISABLE_NON_EI_MODE" or just "ENABLE_EI_MODE"?

Reply:
+1 for ENABLE_EI_MODE

Reply:
If it is crucial that this gets reverted back each time, can we perhaps add a sanity test to ensure that these defaults are as expected?

Reply:
> If it is crucial that this gets reverted back each time, can we perhaps add a sanity test to ensure that these defaults are as expected?

I could not come up with a mechanism for this purpose. I am not sure how to detect if an EI PR is still under review or ready for merging.
If there is a sanity test, do you mean we will fail it during the EI PRs been tested under PR context?

Reply:
@YYStreet yeah, just a sanity test that will fail whenever ENABLE_EI_MODE is set to False. It can be a test called test_build_config (and even test_test_config if you want to implement at the same time). You can just import the global variables and assert their values. This will at least provide another layer to ensure that everyone restores defaults after testing

Reply:
A consistent sanity failure might confuse an away-team contributor. Besides, in the worst case, a PR were merged with this value unreverted, no influence would be introduced for mainline/nightly/release or pipeline. These vars only have effects in PR context.